corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -452,11 +523,17 @@ hqDefine("geospatial/js/geospatial_map", [
         }
 
         $(document).ajaxComplete(function (event, xhr, settings) {
+            const isAfterUserLoad = settings.url.includes('geospatial/get_users_with_gps/');
+            if (isAfterUserLoad) {
+                return;
+            }
```

Comment:
would be good to add a comment on why we are doing this.

Reply:
[9a1c523](https://github.com/dimagi/commcare-hq/pull/33488/commits/9a1c52397c97edab7c831fc760de89c0ab298328)